infra/gcp/ensure-prod-storage.sh

```diff
@@ -235,9 +235,24 @@ empower_artifact_auditor_invoker "${PROD_PROJECT}"
 
 # Special case: empower Kubernetes service account to authenticate as a GCP
 # service account.
+#
+# For write access to k8s-artifacts-prod GCR.
 empower_ksa_to_svcacct \
     "k8s-prow.svc.id.goog[test-pods/k8s-infra-gcr-promoter]" \
     "${PROD_PROJECT}" \
     $(svc_acct_email "${PROD_PROJECT}" "${PROMOTER_SVCACCT}")
+# For write access to:
+#   (1) k8s-gcr-backup-test-prod GCR
+#   (2) k8s-gcr-backup-test-prod-bak GCR.
+# Even though we only grant authentication to 1 SA
+# (k8s-infra-gcr-promoter@k8s-gcr-backup-test-prod-bak.iam.gserviceaccount.com),
+# this SA has write access to the above 2 GCRs, fulfilling our needs.
+#
+# Also, note that the project name for the GKE cluster is "k8s-prow-builds",
+# which is the non-trusted Prow cluster.
+empower_ksa_to_svcacct \
+    "k8s-prow-builds.svc.id.goog[test-pods/k8s-infra-gcr-promoter-test]" \
+    "${PROD_PROJECT}" \
```

Comment:
Shouldn't this be GCR_BACKUP_TEST_PRODBAK_PROJECT ?

Reply:
woops, yes!